class ColumnError(Exception):
    pass


class RowError(Exception):
    pass
